test-cases/performance/perf-loaders-non-shard-aware-config.yaml

```diff
@@ -0,0 +1,16 @@
+# manual on creating loader AMI's: see docs/new_loader_ami.md
+# AMIs from branch-perf-v8 with non-shardware driver
+aws_use_prepared_loaders: true
+regions_data:
+    us-east-1:
```

Comment:
@aleksbykov we are o.k. with this to be only for 3 regions ?

Reply:
Yes, we run performance for that moment only on us-east-1 and sometimes on eu-west-1